From dc0f650ab01a7666e45b40ffd3ddc99001715ffc Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Thu, 25 Jan 2018 19:38:45 +0800
Subject: [PATCH 3270/5242] MLK-17108-1 ARM64: dts: imx8mq-evk: increase i2c1
 clock to be 400K

commit  4295acf3fd458993ba064d6276eb08cefd46a420 from
https://source.codeaurora.org/external/imx/linux-imx.git

As the typec port controller interface(TCPCI) spec requires the i2c clock
at least to be 400K, so here increase it to be 400K to meet timing
requirement.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 434a141..2547352 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -413,7 +413,7 @@
 };
 
 &i2c1 {
-	clock-frequency = <100000>;
+	clock-frequency = <400000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c1>;
 	status = "okay";
-- 
1.7.9.5

